--- ./tv-master/roles/check-latest-minor-version/tasks/main.yml	2019-06-06 13:21:10.696208108 +0200
+++ ./rocky-check-latest-minor-version	2019-06-06 13:21:13.996231214 +0200
@@ -1,16 +1,11 @@
 ---
 - name: Get available updates for packages
-  check_package_update:
-    package: "{{ item }}"
-    pkg_mgr: "{{ ansible_pkg_mgr }}"
+  check_package_update: package={{ item }} pkg_mgr={{ ansible_pkg_mgr }}
   with_items: "{{ packages }}"
   register: updates
 
 - name: Check if current version is latest minor
   with_items: "{{ updates.results }}"
   assert:
     that: "item.latest_minor_version == item.current_version"
-    msg: >-
-      "A newer version of the {{ item.name }} package is
-      available: {{ item.latest_minor_version }} (currently
-      {{ item.current_version }})."
+    msg: "A newer version of the {{ item.name }} package is available: {{ item.latest_minor_version }} (currently {{ item.current_version }})."
