--- ./tv-master/roles/undercloud-selinux-mode/tasks/main.yml	2019-06-03 11:14:34.696966020 +0200
+++ ./rocky-undercloud-selinux-mode	2019-06-03 11:22:39.243485356 +0200
@@ -1,24 +1,25 @@
 ---
-- name:  Get current SELinux mode
-  command: getenforce
-  become: true
-  register: sestatus
-  changed_when: False
+  - name:  Get current SELinux mode
+    command: getenforce
+    become: true
+    register: sestatus
+    changed_when: False
 
-- name: Fail if SELinux is not in Enforced mode (RHEL)
-  fail:
-    msg: >-
-      SELinux is running in {{ sestatus.stdout }} mode on the Undercloud.
-      Ensure that SELinux is enabled and running in Enforcing mode.
-  when:
-    - "sestatus.stdout != 'Enforcing'"
-    - "ansible_distribution == 'RedHat'"
+  - name: Fail if SELinux is not in Enforced mode (RHEL)
+    fail:
+      msg: >-
+        SELinux is running in {{ sestatus.stdout }} mode on the Undercloud.
+        Ensure that SELinux is enabled and running in Enforcing mode.
+    when:
+      - "sestatus.stdout != 'Enforcing'"
+      - "ansible_distribution == 'RedHat'"
+
+  - name: Warn if SELinux is not in Enforced mode (CentOS)
+    warn:
+      msg: >-
+        SELinux is running in {{ sestatus.stdout }} mode on the Undercloud.
+        Ensure that SELinux is enabled and running in Enforcing mode.
+    when:
+      - "sestatus.stdout != 'Enforcing'"
+      - "ansible_distribution == 'CentOS'"
 
-- name: Warn if SELinux is not in Enforced mode (CentOS)
-  warn:
-    msg: >-
-      SELinux is running in {{ sestatus.stdout }} mode on the Undercloud.
-      Ensure that SELinux is enabled and running in Enforcing mode.
-  when:
-    - "sestatus.stdout != 'Enforcing'"
-    - "ansible_distribution == 'CentOS'"
