--- ./tv-master/roles/undercloud-heat-purge-deleted/tasks/main.yml	2019-06-03 11:14:34.696966020 +0200
+++ ./rocky-undercloud-heat-purge-deleted	2019-06-03 11:22:39.202485058 +0200
@@ -1,29 +1,9 @@
 ---
-- name: Get the path of tripleo undercloud config file
-  become: true
-  hiera: name="tripleo_undercloud_conf_file"
-
-- name: Get the Container CLI from the undercloud.conf file
-  become: true
-  ini:
-    path: "{{ tripleo_undercloud_conf_file }}"
-    section: DEFAULT
-    key: container_cli
-    ignore_missing_file: true
-  register: container_cli
-
 - name: Get heat crontab
   become: true
-  shell: |
-    set -o pipefail
-    {{ container_cli.value|default('podman', true) }} exec heat_api_cron crontab -l -u heat |grep -v '^#'
+  shell: 'docker exec heat_api_cron crontab -l -u heat |grep -v "^#"'
   register: cron_result
   changed_when: False
-
 - name: Check heat crontab
-  fail:
-    msg: >-
-      heat-manage purge_deleted does not appear to be enabled via cron. You
-      should add '<desired interval> heat-manage purge_deleted' to the heat
-      users crontab.
-  when: "cron_result.stdout.find('heat-manage purge_deleted') == -1"
+  fail: msg="heat-manage purge_deleted does not appear to be enabled via cron.  You should add '<desired interval > {{ cron_check }}' to the heat users crontab."
+  failed_when: "cron_check not in cron_result.stdout"
