--- ./tv-master/roles/undercloud-cpu/tasks/main.yml	2019-06-06 13:21:10.703208157 +0200
+++ ./rocky-undercloud-cpu	2019-06-06 13:21:14.293233293 +0200
@@ -1,7 +1,4 @@
 ---
 - name: Verify the number of CPU cores
-  fail:
-    msg: >-
-      There are {{ ansible_processor_vcpus }} cores in the system,
-      but there should be at least {{ min_undercloud_cpu_count }}
+  fail: msg="There are {{ ansible_processor_vcpus }} cores in the system, but there should be at least {{ min_undercloud_cpu_count }}"
   failed_when: "ansible_processor_vcpus|int < min_undercloud_cpu_count|int"
